src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/client/BitbucketServerAPIClient.java

```diff
@@ -111,7 +77,49 @@
 import org.apache.http.message.BasicNameValuePair;
 import org.apache.http.util.EntityUtils;
 
-import static java.util.Objects.requireNonNull;
+import com.cloudbees.jenkins.plugins.bitbucket.JsonParser;
```

Comment:
please don't mess with imports or follow the project standard at least.

A-Z, static last with space between.

Reply:
Sorry, was trying to react to the reports, will fix